CesiumIonClient/src/Connection.cpp

```diff
@@ -304,6 +307,34 @@ bool parseJsonObject(const IAssetResponse* pResponse, rapidjson::Document& d) {
 } // namespace
 
 CesiumAsync::Future<Response<Profile>> Connection::me() const {
+  // /v1/me endpoint doesn't exist when ion is running in single user mode
+  if (this->_appData.applicationMode == AuthenticationMode::SingleUser) {
+    Profile profile;
+    profile.id = 0;
+    profile.username = "ion-user";
+    profile.storage = ProfileStorage{
+        0,
+        std::numeric_limits<int64_t>::max(),
+        std::numeric_limits<int64_t>::max()};
+    profile.email = "none@example.com";
+    profile.emailVerified = true;
+    profile.scopes = {
+        "assets:read",
+        "assets:list",
+        "assets:write",
+        "profile:read",
+        "tokens:read",
+        "tokens:write"};
+    profile.avatar = "https://www.gravatar.com/avatar/"
```

Comment:
ion accounts use Gravatar for avatars - it's a service where you can upload an image that gets associated with your email, and other websites can look up your avatar by email and use it. If you don't have an image set it will just give you the default Gravatar avatar, which is the same as you get just requesting `https://www.gravatar.com/avatar/`.